tools/isobuild/compiler-plugin.js

```diff
@@ -91,7 +93,13 @@ export class CompilerPluginProcessor {
     var sourceProcessorsWithSlots = {};
 
     var sourceBatches = _.map(self.unibuilds, function (unibuild) {
+      const { pkg: { name }, arch } = unibuild;
+      const sourceRoot = name
+        ? self.isopackCache.getSourceRoot(name, arch)
```

Comment:
Right, I added `self.isopackCache.getSourceRoot` to abstract over the differences between local/checkout/installed packages. Also note the `name` variable is `null` when we're bundling an app.